server/bleep/src/webserver/answer.rs

```diff
@@ -452,11 +513,20 @@ Answer in GitHub Markdown:",
         prompt
     }
 
-    async fn select_snippet(&self, prompt: &str) -> Result<reqwest::Response, AnswerAPIError> {
-        self.send_until_success(prompt, 1, 0.0).await
+    async fn select_snippet(&self, prompt: &str) -> super::Result<String> {
```

Comment:
We could also maintain the non-streaming `answer-api` endpoint for these single token responses.